<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>857</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simple Scanner</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/images/image/fire.png</normaloff>:/images/image/fire.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QFrame" name="displayFrame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QVTKWidget" name="displayWidget"/>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="commandFrame">
       <property name="minimumSize">
        <size>
         <width>690</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="titlebarLbl">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 104, 221, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Examination Settings</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="settingsContainterTab">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="tabShape">
           <enum>QTabWidget::Triangular</enum>
          </property>
          <property name="currentIndex">
           <number>2</number>
          </property>
          <property name="tabsClosable">
           <bool>false</bool>
          </property>
          <widget class="QWidget" name="energyTab">
           <attribute name="title">
            <string>Energy</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QScrollArea" name="generatorWidgetScrollArea">
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAsNeeded</enum>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QAbstractScrollArea::AdjustToContents</enum>
              </property>
              <property name="widgetResizable">
               <bool>false</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>479</width>
                 <height>384</height>
                </rect>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="detectorTab">
           <attribute name="title">
            <string>Detector</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QScrollArea" name="detectorWidgetScrollArea">
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents_2">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>861</width>
                 <height>436</height>
                </rect>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="actuatorTab">
           <attribute name="title">
            <string>Actuator</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QScrollArea" name="actuatorWidgetscrollArea">
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents_3">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>861</width>
                 <height>332</height>
                </rect>
               </property>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="exposureCommandButtonsFrame">
              <property name="minimumSize">
               <size>
                <width>170</width>
                <height>81</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <layout class="QFormLayout" name="formLayout">
               <item row="0" column="0">
                <widget class="QPushButton" name="startBtn">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>80</width>
                   <height>80</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">
</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="../resources.qrc">
                   <normaloff>:/images/image/start.ico</normaloff>:/images/image/start.ico</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>64</width>
                   <height>64</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="stopBtn">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>80</width>
                   <height>80</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="../resources.qrc">
                   <normaloff>:/images/image/stop.ico</normaloff>:/images/image/stop.ico</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>64</width>
                   <height>64</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowser">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>150</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
background-color: rgb(0, 0, 0);
color: rgb(0, 255, 0);</string>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:7.8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1045</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menuConfiguration">
    <property name="title">
     <string>&amp;Configuration</string>
    </property>
    <addaction name="action_Detector"/>
    <addaction name="action_Generator"/>
    <addaction name="action_Actuators"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuConfiguration"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Detector"/>
   <addaction name="action_Generator"/>
   <addaction name="action_Actuators"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_Detector">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/image/detector.png</normaloff>:/images/image/detector.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Detector</string>
   </property>
  </action>
  <action name="action_Generator">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/image/generator.png</normaloff>:/images/image/generator.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Generator</string>
   </property>
  </action>
  <action name="action_Actuators">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/image/actuator.png</normaloff>:/images/image/actuator.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Actuators</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
